src/scss/abstracts/colors.scss

```
@@ -0,0 +1,19 @@
+// background color:
+$bgColor: #e5e5e5;
+
+// font colors:
+$mainFontColor: #55545a;
+$lighterFontColor: #7e7c82;
+
+// card color:
+$productCardColor: #f5f4f1;
+$productCardColorBlueBorder: #2d6be6;
+$productCardColorRedBorder: #dd333b;
```

```suggestion
$productCardColorSelectedBorder: #2d6be6;
$productCardColorErrorBorder: #dd333b;
```

To nie są dobre nazwy dla zmiennych

To, że ramka jest niebieska to kwestia designu, również dobrze może być zielona czy szara. Jeśli ten kolor się kiedyś zmieni to zamiast zmienić w 1 miejscu wartość, będziesz miał mały refactor. Poza tym, nie wiadomo czytając kod czym jest `RedBorder`, a czym `BlueBorder`.